### DISCUSSION TITLE
pytorch-lighning_version Error

### DISCUSSION THREAD

**Discussion by jganten** [lean-dojo/ReProver#56] [Q&A] [answered]
Hello,
when trying to reproduce the results, I encounter this Error:
`KeyError: 'pytorch-lightning_version'`
This happens when loading the retriever models for corpus indexing.
I have 2.3.0 installed, but I can't find any information, which version is needed.
Your help is appreciated!

**Comment by yangky11** [comment]
Please describe the exact steps for reproducing the error.

**Comment by yangky11** [comment, marked as answer]
I just updated the evaluation script to use Hugging Face checkpoints instead of PyTorch Lightning checkpoints, which hopefully can eliminate this issue. Please follow the instructions in the new README. Feel free to re-open it if you have other questions.
